lib/fetcher.js

```diff
@@ -93,6 +97,12 @@ class FetcherBase {
 
     this.before = opts.before
     this.fullMetadata = this.before ? true : !!opts.fullMetadata
+    this.fullReadJson = !!opts.fullReadJson
```

Comment:
Let's at least add this new opts attribute to the readme

Reply:
Ah. It's already there. We're just ... using it now.

This is a bugfix lol

Reply:
done